### PR TITLE
fix(forms): avoid issues with nulls checking on validation status and…

### PR DESCRIPTION
**Pull request by alxhub**
… other form states.

### CODE REVIEW

File: modules/angular2/src/core/forms/directives/abstract_control_directive.ts [hunk 1]
```diff
@@ -3,17 +3,17 @@ import {AbstractControl} from '../model';
 export class AbstractControlDirective {
   get control(): AbstractControl { return null; }
 
-  get value(): any { return this.control.value; }
+  get value(): any { return this.control != null ? this.control.value : null; }
```
Comment (vsavkin) [inline modules/angular2/src/core/forms/directives/abstract_control_directive.ts]: could you use isPresent(this.control) instead?

**Comment by vsavkin** [comment]
Maybe you can add an integration test that fails without the fix?

File: modules/angular2/src/core/forms/directives/abstract_control_directive.ts (hunk 1 above)
Comment (alxhub) [inline modules/angular2/src/core/forms/directives/abstract_control_directive.ts]: I can do that.

**Comment by alxhub** [comment]
Added an integration test, which does seem to pass. :)

**Pull request closed without merging**